lambda_functions/user_flows/test_retrieve_everything.py

```diff
@@ -38,39 +38,56 @@ def admin_jwt_token(environment) -> str:
 
 @pytest.fixture(scope="session")
 def get_services_response(notification_url, admin_jwt_token) -> Response:
+    """
+    Get an HTTP response with a JSON list of all services.
+    """
+
     return get_authenticated_request(f"{notification_url}/service", admin_jwt_token)
 
 
 @pytest.fixture(scope="session")
 def service_id(get_services_response) -> str:
-    service = next(
-        service for service in get_services_response.json()['data']
-        if service['name'] == "User Flows Test Service"
-    )
-    return service['id']
+    """
+    Return the UUID for the first service with the name "User Flows Test Service".
+    """
+
+    for service in get_services_response.json()['data']:
+        if service['name'] == "User Flows Test Service":
+            return service['id']
+
+    raise RuntimeError("Couldn't find a service ID for User Flows Test Service.")
```

Comment:
```suggestion
    else:
        raise RuntimeError("Couldn't find a service ID for User Flows Test Service.")
```
I'm just kidding, but this is one of the few legit uses of a for/else.

Reply:
I make use of them from time to time.